packages/lib-user/src/components/shared/Layout/Layout.js

```diff
@@ -1,123 +1,87 @@
-'use client'
-
-import { ZooniverseLogotype } from '@zooniverse/react-components'
 import { Box } from 'grommet'
 import { arrayOf, node } from 'prop-types'
 import styled, { css } from 'styled-components'
 
-const PageContainer = styled(Box)`
-  display: grid;
-  grid-template-columns: 0 0 1fr 0 0;
-  grid-template-rows: 0 0 1fr;
-  min-height: 90vh;
+import PageHeader from './components/PageHeader'
 
-  @media (48rem <= width < 80rem) {
-    grid-template-columns: 0 2rem minmax(44rem, 76rem) 2rem 0;
-    grid-template-rows: 50px 50px 1fr;
-  }
-
-  @media (80rem <= width) {
-    grid-template-columns: auto minmax(2rem, 5rem) minmax(74rem, 80rem) minmax(2rem, 5rem) auto;
-    grid-template-rows: 70px 70px 1fr;
-  }
-`
+const InnerPageContainer = styled(Box)`
+  box-shadow: 3px 0px 3px rgba(0, 0, 0, 0.25), -3px 0px 3px rgba(0, 0, 0, 0.25);
 
-const PageLeftHeader = styled(Box)`
-  grid-column: 1 / 3;
-  grid-row: 1 / 3;
-`
+  // Grommet theme's size is definied in pixels
+  // Size "small" is viewport widths less than 769px
+  // If Grommet theme breakpoints are changed to rem units, adjust here as needed
 
-const PageHeader = styled(Box)`
-  grid-column: 3 / 4;
-  grid-row: 1 / 3;
-`
+  @media (768px < width <= 90rem) {
+    padding: 0 30px;
+  }
 
-const PageRightHeader = styled(Box)`
-  grid-column: 4 / 6;
-  grid-row: 1 / 3;
+  @media width <= 769px {
```

Comment:
```suggestion
  @media (width <= 769px) {
```
My linter suggests the parenthesis around the width evaluation, and I notice a different in the inspected CSS, but I don't notice an actual visual difference, so I'm not sure if there's any change to be made here or not.

Reply:
Good point! I'll add parentheses.